src/org/parosproxy/paros/extension/state/ExtensionState.java

```diff
@@ -32,13 +32,21 @@
 // ZAP: 2015/03/16 Issue 1525: Further database independence changes
 // ZAP: 2016/04/05 Issue 2458: Fix xlint warning messages 
 // ZAP: 2016/06/20 Removed unnecessary/unused constructor
+// ZAP: 2016/08/30 Issue 2375: Added option to change ZAP mode in edit menu.
 
 package org.parosproxy.paros.extension.state;
 
 import javax.swing.JCheckBoxMenuItem;
 import javax.swing.JOptionPane;
+import javax.swing.JMenu;
+import javax.swing.event.MenuEvent;

```

Comment:
Unused imports should be removed or not introduced (`MenuEvent`, `MenuListener` and the added `Control.Mode`).